src/Document/CustomBlock.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
```

Comment:
I bet we should add something. But lets leave it to @pjedrzejewski and @michalmarcinkowski